Nighttime-Operations/Observatory/DIMM/index.rst

```diff
@@ -4,11 +4,52 @@
 DIMM
 ###############
 
-This page contains some common calibrations procedures.
+The differential image motion monitor (DIMM) is a standard instrument to measure astronomical "seeing", or atmospheric turbulence strength. 
+
+The DIMM principle is to produce "twin images" of a bright star by a mask with two apertures and a wedge prism at the entrance pupil of the telescope. 
+The rms of the fluctuations of the angular distance between the two images is proportional to the atmospheric optical turbulence, that in turn, allows to predict the seeing FWHM. 
+
+The main control software is divided into several programs. The main services are `ameba` that is in charge of the observation, synchronizing dome, weather service, 
+mount and camera control, and the `preat` program that converts the output of the `dimm_tool` into seeing values.
+
+.. figure:: ./_static/program_structure.png
+    :name: DIMM program structure
+
+    Structure of the DIMM software
+
+
+Below is a list of some of the most useful DIMM manuals:
+
+.. _DIMM General Spec: https://github.com/lsst-ts/ts_dimm_app/blob/develop/doc/pdf/dev-dimm-general_spec-en_V1-1.pdf
+.. _DIMM Configuration: https://github.com/lsst-ts/ts_dimm_app/blob/develop/doc/pdf/dev-dimm-config_spec-en_V1-2.pdf
+.. _tt_master Spec: https://github.com/lsst-ts/ts_dimm_app/blob/develop/doc/pdf/dev-dimm-tt-master_spec-en_V1-4.pdf
+.. _tt_meteo Spec: https://github.com/lsst-ts/ts_dimm_app/blob/develop/doc/pdf/dev-dimm-tt-meteo_spec-en_V1-2.pdf
+.. _Preat Description: https://curl.sai.msu.ru/mass/download/doc/preat_desc2e.pdf 
+.. _DIMM Optical and mechanical design: https://curl.sai.msu.ru/mass/download/doc/description.pdf 
+.. _DIMM soft description: https://curl.sai.msu.ru/mass/download/doc/dimm_soft_description.pdf
```

Comment:
software? 
you'll need to fix where you reference it as well.

Reply:
:) 👍